src/command_modules/azure-cli-interactive/HISTORY.rst

```diff
@@ -3,9 +3,11 @@
 Release History
 ===============
 
-0.3.2 (unreleased)
+0.3.3 (unreleased)
 ++++++++++++++++++
+
 * --progress flag
+* Change the '?' gesture to inject into the next command
```

Comment:
Does it make sense to have both? Or would that make it confusing? Currently, it is a change to the behavior.